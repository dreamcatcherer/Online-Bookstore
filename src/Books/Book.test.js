import React from "react";
import { render } from "@testing-library/react";
import Book from "./Books.jsx";

it("renders the Book component correctly", () => {
  const { container } = render(
    <Book
      categoryId="classics"
      bookId="0001-PrideAndPrejudice"
    />
  );
  expect(container).toMatchSnapshot();
});
