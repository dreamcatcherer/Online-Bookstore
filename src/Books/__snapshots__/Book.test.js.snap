// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Book component correctly 1`] = `
<div>
  <div>
    <center>
      Now Loading...
    </center>
  </div>
</div>
`;
